database_manager/src/run_server.cpp

```diff
@@ -1775,11 +1771,7 @@ void update_world_states(ros::NodeHandle& node, std::vector<ToasterFactReader*>
         for (int i = 0; i < newState.size(); ++i) {
             if ((previousFactsState[j].subjectId == newState[i].subjectId)
                     && (previousFactsState[j].property == newState[i].property)
-                    && (previousFactsState[j].propertyType == newState[i].propertyType)
-                    && (previousFactsState[j].subProperty == newState[i].subProperty)
-                    && (previousFactsState[j].subjectOwnerId == newState[i].subjectOwnerId)
-                    && (previousFactsState[j].targetId == newState[i].targetId)
-                    && (previousFactsState[j].targetOwnerId == newState[i].targetOwnerId)) {
+                    && (previousFactsState[j].targetId == newState[i].targetId)) {
```

Comment:
the problem is that when it is not fill the comparison does not work correctly.
But maybe we can keep at least subProperty